features/resources/being_added_removed_as_a_collaborator_to_from_a_resource.feature

```diff
@@ -0,0 +1,14 @@
+Feature: Being added/removed as a collaborator to/from a resource
+
+Scenario: Being added as a regular single user
+  Given I'm a regular ownCloud user of the instance the resource is located on
+   When I'm added as a collaborator to a resource as in 'Feature: Adding collaborators to a resource'
+   Then the subsequent behavior is determined by 'Feature: Pending Shares'
+
+Scenario: Being removed as a collaborator from a resource
+  Given I'm a collaborator of a certain resource
+   When an Owner removes me from the resource as in 'Scenario: Removing a collaborator from a resource'
+     Or a Co-Owner removes me from the resource as in 'Scenario: Removing a collaborator from a resource'
```

Comment:
Or is not allowed.
You need an extra scenario ;-)

Reply:
IIRC that "Or" thing was introduced by you here: https://github.com/owncloud/product/pull/22/commits/a60caaf7d4b31260a562b59f90144fdadd0b285a#diff-1c40b780bdbf7746a4b6f27bb29fd912R6

Anyway, will try to change it, thanks.